styles.css

```diff
@@ -1,14 +1,81 @@
+/* Globals */
+
+* {
+	box-sizing: border-box;
+}
+
+html {
```

Comment:
As body now has a maximum width, on screens larger than this, when the page scrolls vertically (such as switching to the FAQ tab) the content shifts to accommodate the new scrollbar that appeared.

I think the property `scrollbar-gutter: stable;` placed here in html class will sort this.

Reply:
👍🏻 good spot. All done